app/services/bank_holiday_retriever.rb

```diff
@@ -28,7 +26,7 @@ def response
   end
 
   def uri
-    URI.parse(API_URL)
+    URI.parse(CFEConstants::GOVUK_BANK_HOLIDAY_API_URL)
```

Comment:
I think we should keep what we already have here or maybe it should go in an environment variable.

There's a dependancy between the two constants
API_URL and DEFAULT_GROUP but CFEConstants won't be able to show that clearly.

I wonder if this should be renamed as GovUkBankHolidayRetriever